src/test/resources/gradle-test-project/build.gradle

```diff
@@ -0,0 +1,102 @@
+plugins {
+    id "com.marklogic.ml-gradle" version "4.7.0"
+}
+
+repositories {
+    mavenCentral()
+
+    // Needed to resolve locally-published versions of MLCP.
+    mavenLocal()
+
+    // Needed for some mlcp dependencies, such as commons-csv:1.5.2-marklogic
+    maven { url "https://developer.marklogic.com/maven2/" }
+}
+
+// For MLCP 11.1.0 and higher, Gradle needs to be told which version of the guava dependency to use.
+// Per https://stackoverflow.com/questions/76897459/how-can-i-make-gradle-pick-the-right-guava-variant/77399208#77399208,
+// the following will achieve that.
+configurations {
+    mlcp {
+        attributes {
+            attribute(TargetJvmEnvironment.TARGET_JVM_ENVIRONMENT_ATTRIBUTE, objects.named(TargetJvmEnvironment.class, TargetJvmEnvironment.STANDARD_JVM))
+        }
+    }
+}
+
+dependencies {
+    mlcp "com.marklogic:mlcp:12.0"
```

Comment:
Hi Rob,

Should this be mlcp 11.2?

Reply:
Yes, it should be 11.2. There's likely a way to read it from the pom.xml file, but I think it's going to be changed infrequently enough that it's fine to hard-code it.